apps/app/src/views/routes/NewOrder/components/OrderForm.tsx

```diff
@@ -218,7 +218,10 @@ export const OrderForm = ({
 
         if (
           await confirmWrapper('Send Order?', {
-            description: 'This order will be immediately sent to the pharmacy of choice.',
+            // determine if order is "Send to Patient" by checking if pharmacyId is set
+            description: values.pharmacyId
+              ? 'This order will be sent immediately to the pharmacy of choice.'
+              : "This order will await the patient's choice of pharmacy.",
```

Comment:
"This order will be sent when the patient chooses a pharmacy"

Maybe? 🤷

Reply:
yeah that's better, adding!